app/models/host/base.rb

```diff
@@ -366,7 +366,12 @@ def render_template(template:, **params)
     end
 
     def uptime_seconds
-      self.uptime_fact&.value&.to_i
+      fact = self.uptime_fact
+      if fact&.fact_name&.name == 'proc_stat::btime'
```

Comment:
This is super hacky. Can we please not hardcode subscription manager specific code in core?
Why do we have the fact parser models if we don't use them to parse facts?

Big 👎 from me. Sorry.